cli/package.json

```diff
@@ -35,9 +35,9 @@
   "scripts": {
     "postpack": "rm -f oclif.manifest.json",
     "posttest": "tslint -c ../common/tslint.yml -p . -t stylish",
-    "prepack": "rm -rf lib && tsc -b && oclif-dev manifest && oclif-dev readme",
+    "prepack": "rm -rf lib && tsc && oclif-dev manifest && oclif-dev readme",
     "lint": "tslint -c ../common/tslint.yml -p .",
-    "test": "ava lib/test/**.spec.js",
+    "test": "ava lib/test/**.spec.js lib/test/**.test.js",
```

Comment:
I thought we standardized on `*.spec.*`?

Reply:
Nope, we said, `.test.`, `.spec.` is for E2E or something AFAIR (@arikmaor)?